### DISCUSSION TITLE
blenderBim doc ifcClass

### DISCUSSION THREAD

**Discussion by cadgiru** [IfcOpenShell/IfcOpenShell#2198] [Ideas]
https://blenderbim.org/docs/users/exploring_an_ifc_model.html#viewing-element-classes
![image](https://user-images.githubusercontent.com/42599252/168049193-6a27682d-7cd7-48e6-a5a0-02116acee125.png)
Have had lots of help from https://blenderbim.org/search-ifc-class.html , maybe add link in above warning.. Have also found this page useful for checking ifcClass, in addition to searching

**Comment by Moult** [comment]
Cheers for a bit of closure this has been added a while back but I forgot to respond here
